docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.8"
+services:
+  server:
+    image: klakegg/hugo:0.83.1
```

Comment:
Old Hugo. I find it a very strange that there are no "official" Hugo-image on docker hub...

What do you think @alt-how, is it ok to use this image?

Reply:
It must be updated to use the 0.88.0 version. (There is a PR on 0.88.1 on the maintainer repo).

Reply:
Agree. I suggest using `0.88.0-alpine`, since we're using Alpine as baseline for our other images.

Reply:
This was the suggestion I found in what I believe is the official docs.